backend/battles/templates/battles/battle_list.html

```diff
@@ -1,9 +1,12 @@
 {% load static %}
 
-<link rel="stylesheet" type="text/css" href="{% static 'css/styles.css' %}" />
 <link rel="stylesheet" type="text/css" href="{% static 'css/global.css' %}" />
+<link rel="stylesheet" type="text/css" href="{% static 'css/battle_styles.css' %}" />
 
-<h1>Battles</h1>
+<div class="header">
+  <h1>Battles</h1>
+  <p>{{ request.user }}</p>
```

Comment:
it could be just `{{ user }}``

Reply:
Oh.. and in general, you don't want to "print" the object directly. It's better to be more specific about what you want to display. Eg.: `user.get_full_name()`

Reply:
Will do in a future PR.